pyop2/base.py

```diff
@@ -2786,6 +2747,19 @@ def __iter__(self):
 """
 
 
+class _MapArg(object):
```

Comment:
FWIW, which isn't much, named tuples are slightly slower.  ballpark 300ns per object instantiation as opposed to 250ns.